plugins/MultiSites/Dashboard.php

```diff
@@ -153,19 +164,13 @@ public function getLastDate()
         return $lastPeriod;
     }
 
-    private function convertDataTableToArrayAndApplyFilters(DataTable $table, $request)
+    private function convertDataTableToArrayAndApplyQueuedFilters(DataTable $table, $request)
     {
         $request['serialize'] = 0;
-        $request['expanded'] = 1;
+        $request['expanded'] = 0;
         $request['totals'] = 0;
         $request['format_metrics'] = 1;
-
-        // filter_sort_column does not work correctly is a bug in MultiSites.getAll
-        if (!empty($request['filter_sort_column']) && $request['filter_sort_column'] === 'nb_pageviews') {
-            $request['filter_sort_column'] = 'Actions_nb_pageviews';
-        } elseif (!empty($request['filter_sort_column']) && $request['filter_sort_column'] === 'revenue') {
-            $request['filter_sort_column'] = 'Goal_revenue';
-        }
+        $request['disable_generic_filters'] = 1;
```

Comment:
Generic filters were already executed before, make sure to not execute them again.